### ISSUE TITLE
Not working with Windows

### ISSUE THREAD

**Issue by loganpowell** [supabase/doctest-js#24]
I may be doing something stupid, but I'm getting this even on the README example using jest:

```js
test("should ", () => {
  expect("x").toBe("x")
  doctest("src/sum.js") //?
})
```
results from Jest runner:

```js
 TypeError: doctest is not a function

      3 | test("should ", () => {
      4 |   expect("x").toBe("x")
    > 5 |   doctest("src/sum.js") //?
        |   ^
      6 | })
      7 |
```

**Comment by kiwicopple** [comment]
I haven't actually tried with Jest, but it should work fine. Do you have an existing repo where I can see the code/error (no problem if you don't, I can create).

Just to check, you definitely have `import doctest from '@supabase/doctest-js'`  at the top of your file?

**Comment by loganpowell** [comment]
I think it may be a windows/path issue (i've seen this kind of error before):
```sh
Cannot find module 'C:UsersloganProjects\tutorialsTDDsrcsum.js' 
Require stack: 
- .\node_modules\@supabase\doctest-js\dist\index.js 
- .\src\sum.test.js 
  at ​​​evalExpression​​​ ​./node_modules/@supabase/doctest-js/dist/index.js:571​
  at ​​​exports.default​​​ ​./node_modules/@supabase/doctest-js/dist/index.js:571​
  at ​./node_modules/@supabase/doctest-js/dist/index.js:124​
  at ​​​t.default​​​ ​./node_modules/@supabase/doctest-js/dist/index.js:124​
  at ​​​Object.<anonymous>​​​ ​src/sum.test.js:4:0​
```

here's the code
```js
// src/sum.test.js

import doctest from "@supabase/doctest-js"
import { sum, product, bloop, bleep, blop } from "../lib/sum"

doctest("src/sum.js") //?

test("sum:", () => {
  expect(sum(4, 3)).toBe(7)
})
```

```js
// src/sum.js

/**
 * Returns the sum of 2 numbers
 *
 * @example sum(1, 2)
 * //=> 3
 */
export const sum = (a, b) => {
  return a + b
}

```

**Comment by kiwicopple** [comment]
Aha! Good find. I have a windows machine somewhere - let me dig it up and see if I can fix this

**Comment by soedirgo** [comment]
Closed by #73.
